### ISSUE TITLE
Log transfer and display

### ISSUE THREAD

**Issue by versus666jzx** [streamsync-cloud/streamsync#114]
1. Is it ok that only error logs are transfered into the terminal, but not info and warning?

Code:
``` python
log_type = "error"

initial_state.add_log_entry(
    message=f"{log_type.capitalize()} message",
    type=log_type,
    title=f"This is {log_type} type message"
)
```

Logs in terminal:
![image](https://github.com/streamsync-cloud/streamsync/assets/49030147/aff7e20a-2de1-4d51-82bf-ddd5dd1d1e91)

Log in builder:
![image](https://github.com/streamsync-cloud/streamsync/assets/49030147/041365b2-8ca4-4e30-8931-ed36a41e9c3d)

2. User cant scroll long log message in builder

![image](https://github.com/streamsync-cloud/streamsync/assets/49030147/c9e98352-6f57-47d2-91f9-ff4a49015c12)

Tested on the latest streamsync version - 0.2.7

**Comment by versus666jzx** [comment]
Yes, I use Win 10 and 11 and starting streamsync in PyCharm terminal in venv.

![image](https://github.com/streamsync-cloud/streamsync/assets/49030147/1ea0aecf-9053-4054-b27a-925c9ab456ba)

**Comment by mmikita95** [comment]
After investigation, [this](https://github.com/streamsync-cloud/streamsync/pull/200#issue-2094008559) offers a potential fix. 

@ramedina86 if possible, verification is needed on whether this does not interfere with other existing logging configurations – not in my understanding, but maybe there's some intricacies I'm not yet aware of.

**Comment by ramedina86** [comment]
Fixed in v0.3.0, thanks @versus666jzx for logging this. Took a while but we finally got to it!
